# Chapter 2: Cardinal Points

#### Example 2.1

Given an optical system with a positive focal length of 10 in, find the position and size of the image formed of an object 5-in high which is located 40 in to the left of the first focal point of the system. See Fig 2.5a

![i](imgs/2_5a.png)

Given: f = 10 in, h = 5 in, x = -40 in. Find x' and h'. 

From Newtonian approximation equations
$f\cdot f' = -x\cdot x'$
or

$$x' = \frac{(f\cdot f')}{-x}$$

In [2]:
print("Image distance from 2nd focal point F2 = ", (10*10)/-(-40), "in")

Image distance from 2nd focal point F2 =  2.5 in


Image distance x' = 2.5 in

$$\frac{h}{-h'}=\frac{f'}{x'}$$

Therefore, $$h' = \frac{-(h\cdot x')}{f'}$$

In [4]:
print("Image height h' = ", -(5*(2.5))/10, "in")

Image height h' =  -1.25 in


Image height h' = -1.25 in (inverted)

$$f'(a) = \lim_{x \to a} \frac{f(x) - f(a)}{x-a}$$

$f'(a) = \lim_{x \to a} \frac{f(x) - f(a)}{x-a}$